recipes/ua-nodeset/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class UaNodeSetConan(ConanFile):
+    name = "ua-nodeset"
+    license = "MIT"
+    description = "UANodeSets and other normative files which are released with a specification"
+    homepage = "https://github.com/OPCFoundation/UA-Nodeset"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("opc-ua-specifications", "uanodeset", "normative-files", "companion-specification")
+
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _extract_license(self):
+        content = tools.load(os.path.join(self.source_folder, self._source_subfolder, "AnsiC", "opcua_clientapi.c"))
```

Comment:
I see there are some sources (`opcua_clientapi.c` at very least), but I don't see anything built/compiled in the conanfile.
there is nothing included or linked in the test package as well, it only checks for some `PLCopen` file.
could you explain how this project is exactly used?

Reply:
These source files are just example files which could be used in combination with the official OPC Foundation reference ANSI C implementation, which is not open source.

The other files, especially the NodeSet2.xml files (together with their Types.xsd and NodeId.csv) files are machine-readable descriptions of an OPC UA companion specification.

The open62541 stack includes the nodeset generator, which generates C Code from these XML files, to correctly initialize an OPC UA Server with a given companion specification.

That script is added in #7314 to open62541.
So open62541 depends on ua-nodeset if one wants to create a pre-initialized OPC UA server.
ua-nodeset conan package is just delivering the resource files, and no libraries or binaries.